package.json

```diff
@@ -94,5 +94,10 @@
 	"resolutions": {
 		"js-beautify": "1.7.5",
 		"event-stream": "3.3.4"
+	},
+	"dependencies": {
+		"immutability-helper": "^3.0.1",
+		"react-dnd": "^10.0.2",
+		"react-dnd-html5-backend": "^10.0.2"
```

Comment:
When doing `yarn add react-dnd react-dnd-html5-backend` in the demos package I get this error:
`error An unexpected error occurred: "Cannot read property 'manifest' of undefined".`. That's why I added them to the root package.

Reply:
I think the real reason is one line above that message:

```
warning Missing version in workspace at "/Users/greghurrell/code/clay/packages/demos", ignoring.
error An unexpected error occurred: "Cannot read property 'manifest' of undefined".
```

This is happening because there is no `version` field in `packages/demos/package.json`. Just add a bogus one (eg. "0.0.1") and it you will be able to add the dependencies locally.

Reply:
@wincent interesting how the line that it ignores is the one that causes the error. I thought about doing that but assumed that the version was intentionally omitted. Works now, thanks!